jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/openshift/RouteEnricher.java

```diff
@@ -50,11 +49,16 @@
 public class RouteEnricher extends BaseEnricher {
 
     private static final String GENERATE_ROUTE_PROPERTY = "jkube.openshift.generateRoute";
+    private static final String GENERATE_TLS_TERMINATION_PROPERTY = "jkube.openshift.generateRoute.tls.termination";
+    private static final String GENERATE_TLS_INSECURE_EDGE_TERMINATION_POLICY_PROPERTY = "jkube.openshift.generateRoute.tls.insecure_edge_termination_policy";
```

Comment:
Unless there is a specific reason to include custom properties (e.g. compatibility with other systems, libraries, etc.) the recommended approach is to use the built-in property & configuration system.

In this case, both your configurations are configurable by the use of the following properties:
```
jkube.enricher.jkube-openshift-route.termination
jkube.enricher.jkube-openshift-route.insecureEdgeTerminationPolicy
```

Please, if there is no reason to add this extra complexity, remove these properties and make use of the standard approach.

Reply:
Of course - was not aware that these existed. Can you point me to an overview of existing properties? Can't seem to find these in the project

Reply:
@manusa  now I think I get your point - follow the pom layout for property naming? 
In that case, I suppose also the GENERATE_ROUTE_PROPERTY should be jkube.enricher.jkube-openshift-route.generateRoute?

Reply:
Yep, you can check that in the docs (Line 228):
https://github.com/eclipse/jkube/blob/7e42f3bad9bec43d82a2425e89d9b1dc94408c8a/kubernetes-maven-plugin/doc/src/main/asciidoc/inc/goals/build/_jkube-resource.adoc#L223-L228

You **don't** need to provide any property resolution yourself, `getConfig` method takes care of that for you.

The other property has been kept for backwards compatibility with FMP, but this is not what we want for new properties.